### DISCUSSION TITLE
How to call python script from exec component.

### DISCUSSION THREAD

**Discussion by satscreate** [vectordotdev/vector#13342] [Q&A] [answered]
Hi, I used to call python script using below lines with custom vector linux image. This is working under docker perspective.
`command: ["bash", "-c", "python script.py"]`

Now how can i run this in windows command line using vector installed,
**CMD versions:**
```
C:\Users\test>vector --version
vector 0.20.1 (x86_64-pc-windows-msvc 38b806e 2022-04-07)
```

```
C:\Users\test>python -V
Python 3.9.1
```
```
C:\Users\test>python script.py
Running....
```
```
C:\Users\test>vector -c test.yaml
/bin/bash: line 1: python: command not found" internal_log_rate_secs=1 vrl_position=76
```

Any way i get it running in local windows machine?

**Edit - FYI,**
It does work with below config: Is this the only way?
`command: ["C:/Python39/python.exe","C:/Users/test/script.py"]`

**Comment by jszwedko** [comment, marked as answer]
Hi @satscreate !

It seems like `python` isn't in the path when running via `bash`. In your example, what happens if you run `bash` and then `which python`? My assumption is that it won't be found.

If you don't want to provide a fully qualified path, you'll want to ensure that the python binary is in your environment path. See https://superuser.com/questions/284342/what-are-path-and-other-environment-variables-and-how-can-i-set-or-use-them for more details.
